livekit-agents/livekit/agents/cli/protocol.py

```diff
@@ -56,7 +57,8 @@ def read(self, b: io.BytesIO) -> None:
             job.ParseFromString(job_s)
             accept_s = ipc_enc._read_bytes(b)
             accept_data = pickle.loads(accept_s)
-            self.jobs.append(ActiveJob(job=job, accept_data=accept_data))
+            token = ipc_enc._read_string(b)
```

Comment:
@theomonnom please verify my pickle logic. I didn't do a deep dive into the ipc_enc workings.